scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -929,14 +929,7 @@
   (with-redefs [completion/plugin completion/no-op]
     (let [uri "datomic:mem://test-handle-status-update"
           conn (restore-fresh-database! uri)
-          tasks-killed (atom #{})
-          driver (reify msched/SchedulerDriver
-                   (kill-task! [_ task] (swap! tasks-killed conj (:value task)))) ; Conjoin the task-id
-          compute-cluster (testutil/fake-test-compute-cluster-with-driver conn uri driver)
-          fenzo (sched/make-fenzo-scheduler compute-cluster 1500 nil 0.8)
-          synced-agents-atom (atom [])
```

Comment:
The synced-agents-atom and synced-agent-sandboxes-fn machinery to update it is dead and unchecked. Shall I open a followup to resurrect it?

Reply:
I'm guessing you mean the machinery in the test is dead? If so, do we have any other tests for this functionality?

Reply:
Correct. Don't know. I suspect not. (at least at the unit test level)